### DISCUSSION TITLE
animation not showing on page

### DISCUSSION THREAD

**Discussion by bigbadbrad** [bibixx/react-adobe-animate#64] [Q&A / Help] [answered]
Hi,
I have the example running locally.  I added an elephant animation which comes with Animate.  I exported it as html canvas and added it to public.  I updated index with: <script src="./elephant.js" type="text/javascript"></script>
and I call it in App.js: 
     <AnimateCC
        animationName="elephant"
        getAnimationObject={getAnimationObject}
        paused={paused}
[animations.zip](https://github.com/bibixx/react-adobe-animate/files/6971559/animations.zip)

      />
But it isn't rendering.  I read the documentation but cannot figure out what I'm missing.  Attached are the assets.  thx!

**Comment by bibixx** [comment]
🤔 It seems as though Adobe has changed the output file format yet again. What version of Adobe Animate have you used to generate the JS and html files?

**Comment by bibixx** [comment, marked as answer]
Move the discussion to #65
